src/HeadProvider.js

```diff
@@ -12,13 +14,19 @@ export default class HeadProvider extends React.Component {
 
   state = {
     list: [],
-    addClientTag: tag => {
+    addClientTag: (tag, name) => {
       this.setState(state => ({
-        list: [...state.list, tag],
+        list: [...state.list, `${tag}:${name}`],
```

Comment:
Maybe an edge case, but what about if `:` is in the name? e.g.

```
<meta name="foo:bar">baz</meta>
```

Maybe could store this as an associative array instead?